tokio/src/fs/file.rs

```diff
@@ -210,7 +210,7 @@ impl File {
         });
 
         new_instrumented_resource! {
-            FileSystem,
+            "fileSystem",
```

Comment:
nit/tioli: i might call this
```suggestion
            "fs",
```
for consistency with the module naming (and it's a little more concise but i think people will know what it means)